packages/orchestration/test/ibc-mocks.ts

```diff
@@ -14,47 +23,92 @@ import {
   buildQueryPacketString,
 } from '../tools/ibc-mocks.js';
 
-const responses = {
-  delegate: buildMsgResponseString(MsgDelegateResponse, {}),
-  queryBalance: buildQueryResponseString(QueryBalanceResponse, {
-    balance: { amount: '0', denom: 'uatom' },
-  }),
+/**
+ * TODO: provide mappings to cosmos error codes (and module specific error codes)
+ * see https://github.com/Agoric/agoric-sdk/issues/9629 for more details about
+ * error messages over ibc
+ */
+export const errorAcknowledgments = {
   error5: buildMsgErrorString(
     'ABCI code: 5: error handling packet: see events for details',
   ),
 };
 
+const delegation = {
+  amount: {
+    denom: 'uatom',
+    amount: '10',
+  },
+  delegatorAddress: 'cosmos1test',
+  validatorAddress: 'cosmosvaloper1test',
+};
+const redelegation = {
+  delegatorAddress: 'cosmos1test',
+  validatorSrcAddress: 'cosmosvaloper1test',
+  validatorDstAddress: 'cosmosvaloper2test',
+  amount: {
+    denom: 'uatom',
+    amount: '10',
+  },
+};
+
+export const UNBOND_PERIOD_SECONDS = 5n;
+
+/**
+ * returns Timestamp record 5 seconds from unix epoch.
+ */
+const getCompletionTime = (): Timestamp => {
+  return {
+    seconds: UNBOND_PERIOD_SECONDS,
+    nanos: 0,
+  };
+};
```

Comment:
it's only five because of `UNBOND_PERIOD_SECONDS`. if someone changes that they won't know to change this comment.

the 0+ is thoughtful, but I don't think the "since unix epoch" is relevant to the test

please simplify and make more explicit what it gets
```suggestion
const getUnbondingTime = () => new Date(Number(UNBOND_PERIOD_SECONDS * MILLISECONDS_PER_SECOND));
```

Reply:
Simplified but kept the return time a Timestamp

Reply:
oh good. I made the suggestion on an earlier commit and when I pulled it to the PR level I forgot to adjust the return type